src/toolbox.ts

```diff
@@ -30,77 +31,84 @@ enum ValidResizeValues {
+const toolboxDividerDiv = "<div class=toolbox-divider></div>";
 
 /** Chains the replaceAll method and the toLowerCase method.
  *  Optionally concatenates a string at the end of the method.
-  */
-String.prototype.replaceLowerConcat = function(before: string, after: string, concat_string: string = null) {
-    
-    if (typeof(concat_string) === "string") {
-        return this.replaceAll(before, after).toLowerCase().concat(concat_string)
+ */
+String.prototype.replaceLowerConcat = function (
+    before: string,
+    after: string,
+    concat_string: string = null,
+) {
+    if (typeof concat_string === "string") {
+        return this.replaceAll(before, after)
+            .toLowerCase()
+            .concat(concat_string);
     }
 
-    return this.replaceAll(before, after).toLowerCase()
-}
-
+    return this.replaceAll(before, after).toLowerCase();
+};
 
 /**
  * Manager for toolbox. Contains ToolboxTab items.
  */
 export class Toolbox {
-    
     constructor(
         public tabs: ToolboxTab[] = [],
-        public items: ToolboxItem[] = []
-    ) {
-    }
+        public items: ToolboxItem[] = [],
+    ) {}
 
-    public static create_toolbox(ulabel: ULabel, toolbox_item_order: unknown[]) {
+    public static create_toolbox(
+        ulabel: ULabel,
+        toolbox_item_order: unknown[],
+    ) {
         // Grab the default toolbox if one wasn't provided
         if (toolbox_item_order == null) {
-            toolbox_item_order = ulabel.config.toolbox_order
+            toolbox_item_order = ulabel.config.toolbox_order;
         }
 
         // There's no point to having an empty toolbox, so throw an error if the toolbox is empty.
         // The toolbox won't actually break if there aren't any items in the toolbox, so this
         // error isn't strictly neccesary.
         if (toolbox_item_order.length === 0) {
-            throw new Error("No Toolbox Items Given")
+            throw new Error("No Toolbox Items Given");
         }
 
-        this.add_styles()
+        this.add_styles();
 
-        let toolbox_instance_list = [];
+        const toolbox_instance_list = [];
         // Go through the items in toolbox_item_order and add their instance to the toolbox instance list
         for (let i = 0; i < toolbox_item_order.length; i++) {
-
             let args: object, toolbox_key: number;
 
-            // If the value of toolbox_item_order[i] is a number then that means the it is one of the 
+            // If the value of toolbox_item_order[i] is a number then that means the it is one of the
             // enumerated toolbox items, so set it to the key, otherwise the element must be an array
             // of which the first element of that array must be the enumerated value, and the arguments
             // must be the second value
-            if (typeof(toolbox_item_order[i]) === "number") {
-                toolbox_key = <number> toolbox_item_order[i]
+            if (typeof toolbox_item_order[i] === "number") {
```

Comment:
The way this was originally written was unnecessary; `typeof` doesn't need parentheses like a function call. 

However, if you wrap the whole of the left clause in parentheses:
```typescript
if ((typeof toolbox_item_order[i]) === "number") {
```
`prettier` will remove them. 

ESLint and "Typescript Language Features" don't care. I mainly come from the python world, so I rarely see excess parentheses in statements. I'm leaning towards `prettier` being correct in this case. Thoughts?